app/slot_ticker.go

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2022-2023 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package app
+
+import (
+	"context"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/eth2wrap"
+	"github.com/obolnetwork/charon/app/log"
+	"github.com/obolnetwork/charon/app/z"
+	"github.com/obolnetwork/charon/core"
+)
+
+// newSlotTicker returns a blocking channel that will be populated with new slots in real time.
+// It is also populated with the current slot immediately.
+//
+// Taken from Charon's core/scheduler package.
```

Comment:
can you add a permalink to that section in charon?

Reply:
hmmm i can but that link might get stale fast if we end up refactoring scheduler, and result in 404 :/